From 0e08c9ec4c2d12f6870a701010bf8e0c5053a974 Mon Sep 17 00:00:00 2001
From: Leandro Lanzieri <leandro.lanzieri@haw-hamburg.de>
Date: Fri, 17 Oct 2025 13:44:37 +0200
Subject: [PATCH] fix void prototypes

---
 lib/Distribution/embedDB.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/lib/Distribution/embedDB.h b/lib/Distribution/embedDB.h
index 0ab3aaa..750aa32 100644
--- a/lib/Distribution/embedDB.h
+++ b/lib/Distribution/embedDB.h
@@ -598,7 +598,7 @@ int8_t embedDBInit(embedDBState *state, size_t indexMaxError);
  * @brief	Initialize embedDB structure with default parameters.
  * @return	Returns an embedDB state with default values if successful, and NULL if not
  */
-embedDBState *defaultInitializedState();
+embedDBState *defaultInitializedState(void);
 
 /**
  * @brief   Prints the initialization stats of the given embedDB state
@@ -1050,7 +1050,7 @@ embedDBOperator *createKeyJoinOperator(embedDBOperator *input1, embedDBOperator
 /**
  * @brief	Creates an aggregate function to count the number of records in a group. To be used in combination with an embedDBOperator produced by createAggregateOperator
  */
-embedDBAggregateFunc *createCountAggregate();
+embedDBAggregateFunc *createCountAggregate(void);
 
 /**
  * @brief	Creates an aggregate function to sum a column over a group. To be used in combination with an embedDBOperator produced by createAggregateOperator. Column must be no bigger than 8 bytes.
-- 
2.51.0

